softlayer/resource_softlayer_virtual_guest.go

```diff
@@ -270,6 +270,19 @@ func resourceSoftLayerVirtualGuest() *schema.Resource {
 				Computed: true,
 			},
 
+			"secondary_ip_count": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				ForceNew: true,
+				Default:  0,
```

Comment:
int type doesn't need to specify default of 0 I believe. That is the type's default already.